ntbs-service/Migrations/20190923171506_AddNotificationCreationDate.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using Microsoft.EntityFrameworkCore.Migrations;
+
+namespace ntbs_service.Migrations
+{
+    public partial class AddNotificationCreationDate : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.AddColumn<DateTime>(
+                name: "CreationDate",
+                table: "Notification",
+                nullable: false,
+                defaultValue: new DateTime(1, 1, 1, 0, 0, 0, 0, DateTimeKind.Unspecified));

```

Comment:
Currently I'm setting the creation date in Create.cshtml.cs - I could remove this and just let sql have the default value of now (or both), what do people think is best?

Reply:
I prefer setting it in the application, as it keeps the logic in one place and doesn't rely on the timezone of the SQL server being correct (which, aparently, is not something we want to rely on :/ )